data/mods/Aftershock/npcs/factions.json

```diff
@@ -91,7 +92,8 @@
       { "item": "UICA_1000d", "fixed_adj": 0 },
       { "item": "UICA_10000d", "fixed_adj": 0 }
     ],
-    "fac_food_supply": { "calories": 115200, "vitamins": {  } },
+    "//": "90 days worth of food for 10 people, at 3000kcal/day and full RDA of iron/calcium/vitC",
+    "fac_food_supply": { "calories": 2700000000, "vitamins": { "iron": 86400, "calcium": 86400, "vitC": 86400 } },
```

Comment:
UICA should probably have infinite resources of most kinds. With the food regen it isn't a blocker now, but its worth considering a possible solution while the design isnt yet finalized.

Reply:
There's an interesting 'problem' here in that calories is a 64-bit integer but vitamins is a 32-bit integer. If we were to use INTMAX for both of them the effective vitamins per day would actually go down, falling far below RDA because int64 just has so many more zeros at the end.

I've put in 90% of the max for a 32-bit integer and a little note, though.